.github/actions/spell-check/whitelist.txt

```diff
@@ -0,0 +1,3240 @@
+CURRENTPAGE
+CURSORCOLOR
+CURSORSIZE
+CURSORTYPE
+customframe
+CUU
+cvd
+cw
+Cwa
+cwch
+cwchar
+cwctype
+cwd
+cx
+CXFRAME
+CXFULLSCREEN
+CXHSCROLL
+CXMIN
+CXPADDEDBORDER
+CXSIZE
+CXSMICON
+CXVIRTUALSCREEN
+CXVSCROLL
+cxx
+CYFRAME
+CYFULLSCREEN
+cygwin
+CYHSCROLL
+CYMIN
+CYPADDEDBORDER
+CYSIZE
+CYSIZEFRAME
+CYSMICON
+CYVIRTUALSCREEN
+CYVSCROLL
+DADADA
+dahall
+dai
+DATABLOCK
+datacollector
+dataprotection
+dataprotectionprovider
+DATAVIEW
+DATAW
+Datei
+Dateien
+dba
+DBatch
+dbb
+dbcs
+DBCSCHAR
+DBCSFONT
+dbd
+DBDCF
+DBDDE
+dbf
+DBFFu
+dbg
+DBGALL
+DBGCHARS
+DBGFONTS
+DBGOUTPUT
+dbh
+dblclk
+DBlob
+dbmdl
+dbproj
+DBUILD
+DCDFE
+dcf
+DColor
+DCompile
+dcompiler
+ddb
+dde
+DDESHARE
+DDevice
+DEADCHAR
+dealloc
+debian
+debolden
+debounce
+DECALN
+DECANM
+DECAUPSS
+DECAWM
+DECCKM
+DECCOLM
+decf
+DECKPAM
+DECKPM
+DECKPNM
+DECLL
+DECLRMM
+decls
+declspec
+decltype
+DECNKM
+DECNRCM
+DECOM
+deconstructed
+DECPCTERM
+DECRC
+DECRLM
+DECRST
+DECSASD
+DECSC
+DECSCA
+DECSCNM
+DECSCPP
+DECSCUSR
+DECSED
+DECSEL
+DECSET
+DECSLRM
+DECSMBV
+DECSR
+decstandar
+DECSTBM
+DECSTR
+DECTCEM
+Dedupe
+deduplicated
+DEFAULTBACKGROUND
+DEFAULTFOREGROUND
+defaultmk
+defaultsettings
+DEFAULTTONEAREST
+DEFAULTTONULL
+DEFAULTTOPRIMARY
+defectdefs
+DEFERERASE
+deff
+DEFFACE
+defing
+DEFPUSHBUTTON
+DELAYLOAD
+deletable
+DELETEONRELEASE
+delims
+Delt
+demoable
+depersist
+deprioritized
+deque
+deref
+deserialization
+deserialize
+deserialized
+deserializer
+deserializing
+desktopwindowxamlsource
+dest
+DESTINATIONNAME
+devblogs
+developercommunity
+devicecode
+devicefamily
+devops
+Dext
+df
+dfa
+DFactory
+DFF
+DFFCAC
+DFFF
+DFFFu
+DFu
+DGCZZu
+dh
+dhowett
+dialogbox
+diesem
+diffing
+DINLINE
+directio
+directx
+dirs
+DISABLENOSCROLL
+DISOLATION
+DISPLAYATTRIBUTE
+DISPLAYATTRIBUTEPROPERTY
+DISPLAYCHANGE
+distro
+Diviness
+dlg
+DLGC
+dll
+dlldata
+DLLDEF
+DLLENTRY
+dllexport
+DLLGETVERSIONPROC
+dllimport
+dllinit
+DLLVERSIONINFO
+DLOAD
+DLOOK
+DNT
+docstates
+DOCTYPE
+docx
+DONTCARE
+doomo
+doskey
+dotnet
+doubleclick
+downlevel
+dpg
+dpi
+DPIAPI
+DPICHANGE
+DPICHANGED
+dpix
+dpiy
+draggable
+DRAWFRAME
+DRAWITEM
+DRAWITEMSTRUCT
+dropdown
+DROPDOWNLIST
+DROPFILES
+drv
+dsm
+Dst
+DSTRICT
+DSwap
+DSYSTEM
+DTest
+dtor
+DTTERM
+duhowett
+DUMMYUNIONNAME
+DUNICODE
+DUNIT
+dup'ed
+dvi
+dw
+DWIN
+DWLP
+dwm
+dwmapi
+dword
+dwrite
+dwriteglyphrundescriptionclustermap
+dxgi
+dxgidwm
+dxinterop
+dxttbmp
+DYNLINK
+eachother
+EAEC
+eaf
+EASTEUROPE
+eb
+eba
+ebce
+EBFB
+ebug
+ECEAF
+ecf
+ECFB
+ECH
+echokey
+ecount
+ECpp
+ectread
+EDFF
+edist
+EDITKEYS
+Editses
+EDITTEXT
+EDITUPDATE
+edputil
+edu
+eeb
+eee
+EEF
+EEu
+Efast
+EFB
+EFBC
+EFDAC
+EFu
+egistry
+EHsc
+ein
+EJO
+EK
+EKU
+elease
+ELEMENTNOTAVAILABLE
+elemetry
+Elems
+elif
+elseif
+emacs
+emoji
+emplates
+emptybox
+enabledelayedexpansion
+enderer
+endian
+endif
+endl
+endlocal
+endptr
+endregion
+ENQ
+enqueuing
+entrypoint
+ENU
+enum
+ENUMLOGFONT
+ENUMLOGFONTEX
+enumranges
+envvar
+eol
+ERASEBKGND
+erm
+erminal
+erminalcore
+erminalinput
```

Comment:
ooof, this is painful. `erminal` for `terminal`.

Maybe if we can get the multiple files thing, we can have a `whitelist_t.txt` that has all the words where t is cutoff (and so on?)